packages/google_sign_in/google_sign_in_web/example/integration_test/gapi_utils_test.dart

```diff
@@ -7,21 +7,16 @@ import 'package:integration_test/integration_test.dart';
 
 import 'package:google_sign_in_web/src/generated/gapiauth2.dart' as gapi;
 import 'package:google_sign_in_web/src/utils.dart';
-import 'package:mockito/mockito.dart';
-
-class MockGoogleUser extends Mock implements gapi.GoogleUser {}
-
-class MockBasicProfile extends Mock implements gapi.BasicProfile {}
 
 void main() {
   // The non-null use cases are covered by the auth2_test.dart file.
   IntegrationTestWidgetsFlutterBinding.ensureInitialized();
 
   group('gapiUserToPluginUserData', () {
-    var mockUser;
+    late FakeGoogleUser mockUser;
```

Comment:
nit: `fakeUser` since it's not doing any validation.

Reply:
Done